File: .skills/openclaw-skills/skills/issa-me-sush/openserv-agent-sdk/examples/basic-agent.ts
/**
 * Basic Agent Example
 *
 * A minimal agent setup using @openserv-labs/sdk and @openserv-labs/client.
 * Just define, provision, and run!
 *
 * Run with: npx tsx basic-agent.ts
 */
import 'dotenv/config'
import { Agent, run } from '@openserv-labs/sdk'
import { provision, triggers } from '@openserv-labs/client'
import { z } from 'zod'

// 1. Define your agent
const agent = new Agent({
  systemPrompt: 'You are a helpful assistant that greets users.'
})

// 2. Add capabilities
agent.addCapability({
  name: 'greet',
  description: 'Greet a user by name',
  inputSchema: z.object({
    name: z.string().describe('The name of the user to greet')
  }),
  async run({ args }) {
    console.log(`Greeting: ${args.name}`)
    return `Hello, ${args.name}! Welcome to OpenServ.`
  }
})

async function main() {
  // 3. Provision with agent instance binding (v2.1+)
  // Binds API key and auth token directly to agent - no env vars needed!
  const result = await provision({
    agent: {
      instance: agent, // Calls agent.setCredentials() automatically
      name: 'basic-greeter',
      description: 'A simple greeting agent'
    },
    workflow: {
      name: 'Welcome Wizard',
      goal: 'Welcome users by name with a warm, personalized greeting message',
      trigger: triggers.webhook({
        waitForCompletion: true,
        input: {
          name: { type: 'string', title: 'Your Name', description: 'Who should we greet?' }
        }
      }),
      task: { description: 'Greet the user' }
    }
  })

  console.log(`Webhook: https://api.openserv.ai/webhooks/trigger/${result.triggerToken}`)

  // 4. Run the agent - credentials already bound via provision()
  await run(agent)
}

main().catch(console.error)

File: .skills/openclaw-skills/skills/sendwealth/openai-agent-sdk/examples/basic_agent.py
"""
基础 Agent 示例
演示如何创建和运行一个简单的 Agent
"""

import asyncio
from agents import Agent, Runner


async def main():
    # 创建一个简单的 Agent
    agent = Agent(
        name="助手",
        instructions="你是一个有帮助的助手，用中文回答问题。",
    )

    # 运行 Agent
    result = await Runner.run(agent, "介绍一下你自己")
    
    print("=" * 50)
    print("Agent 回复：")
    print("=" * 50)
    print(result.final_output)
    print("=" * 50)


if __name__ == "__main__":
    asyncio.run(main())
